README.md

```diff
@@ -6,19 +6,19 @@ This repository provides a [Terraform module](https://learn.hashicorp.com/tutori
 
 This repository provides four submodules:
 
```

Comment:
strange we went from 5.6 to 5.8 - we missed this in 5.7? can we make sure to add this step somewhere so we don't forget again? :)

Reply:
Its in our run book? Not sure why it was missed or how we patched the thing

Reply:
Sounds like we may have just forgot to merge the change